4-Soru.py

```diff
@@ -0,0 +1,19 @@
+from collections import OrderedDict
+
+def removeDupWithoutOrder(str):
+
+    str = ''.join(str.split())

```

Comment:
Bu satirdaki kod ile cumlenin icindeki bosluklari yok etmis oluyorsunuz. Soruda boslugu da dikkate almaniz istenmisti.

Reply:
`str` Python'in kendi kullandigi bir data type ismi. Degisken isimleri kullanirken str, list, dict gibi data type isimlerini kullanmamaniz tavsiye edilir.